src/index.js

```diff
@@ -0,0 +1,4 @@
+import './css/style.css'
+import Tabs from './classes/Tabs'
```

Comment:
Codacy has a fix for the issue: [Replace `Tabs·from·'./classes/Tab` with `'./css/style.cs`](https://app.codacy.com/gh/BeAPI/accessible-components/pullRequest?prid=8000264)
```suggestion
import './css/style.css'
```